### PR TITLE
Note about PSR-15 middleware

### PR DESCRIPTION
**Pull request by WyriHaximus**
With the acceptance of PSR-15 we should add a note that we support for it through the FoR adapter

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1129,6 +1130,10 @@ new RequestBodyParserMiddleware(10 * 1024, 100); // 100 files with 10 KiB each
 
 A non-exhaustive list of third-party middleware can be found at the [`Middleware`](https://github.com/reactphp/http/wiki/Middleware) wiki page. 
 
+#### PSR-15 Middleware
+
+Additionally we support PSR-15 through the [Friends of ReactPHP](https://github.com/friends-of-reactphp) [PSR-15 middleware adapter](https://github.com/friends-of-reactphp/http-middleware-psr15-adapter).
```
Comment (jsor) [inline README.md]: I'd remove "Additionally" as the reference to the previous section isn't required and probably confusing.

Comment (WyriHaximus) [reply README.md]: @jsor Good one, updated the PR

**Comment by WyriHaximus** [comment]
@clue makes sense, I'll add a bit about why we don't support it directly in `react/http` :+1: . (The answer is really simple though, return types.)

**Comment by WyriHaximus** [comment]
@clue reworded the PSR-15 paragraph and the Middleware chapter where it mentions the `ongoing` effort

**Pull request closed without merging**